code/modules/mob/living/carbon/human/emote.dm

```diff
@@ -826,34 +826,34 @@
 				return
 //			playsound(loc, 'sound/effects/fart.ogg', 50, 1, -3) //Admins still vote no to fun
 			if(locate(/obj/item/storage/bible) in get_turf(src))
-				to_chat(viewers(src), "<span class='warning'><b>[src] farts on the Bible!</b></span>")
-				var/image/cross = image('icons/obj/storage.dmi',"bible")
-				var/adminbfmessage = "\blue [bicon(cross)] <b><font color=red>Bible Fart: </font>[key_name(src, 1)] (<A HREF='?_src_=holder;adminmoreinfo=\ref[src]'>?</A>) (<A HREF='?_src_=holder;adminplayeropts=\ref[src]'>PP</A>) (<A HREF='?_src_=vars;Vars=[UID()]'>VV</A>) (<A HREF='?_src_=holder;subtlemessage=\ref[src]'>SM</A>) ([admin_jump_link(src)]) (<A HREF='?_src_=holder;secretsadmin=check_antagonist'>CA</A>) (<A HREF='?_src_=holder;Smite=[UID()]'>SMITE</A>):</b>"
+				to_chat(viewers(src), "<span class='danger'>[src] farts on the Bible!</span>")
+				var/image/cross = image('icons/obj/storage.dmi', "bible")
+				var/adminbfmessage = "[bicon(cross)] <span class='danger'>Bible Fart:</span> [key_name(src, 1)] (<A HREF='?_src_=holder;adminmoreinfo=[UID()]'>?</A>) (<A HREF='?_src_=holder;adminplayeropts=[UID()]'>PP</A>) (<A HREF='?_src_=vars;Vars=[UID()]'>VV</A>) (<A HREF='?_src_=holder;subtlemessage=[UID()]'>SM</A>) ([admin_jump_link(src)]) (<A HREF='?_src_=holder;secretsadmin=check_antagonist'>CA</A>) (<A HREF='?_src_=holder;Smite=[UID()]'>SMITE</A>):</b>"
 				for(var/client/X in admins)
-					if(check_rights(R_EVENT,0,X.mob))
+					if(check_rights(R_EVENT, 0, X.mob))
 						to_chat(X, adminbfmessage)
 			else if(TOXIC_FARTS in mutations)
-				message = "<b>[src]</b> unleashes a [pick("horrible","terrible","foul","disgusting","awful")] fart."
+				message = "<b>[src]</b> unleashes a [pick("horrible", "terrible", "foul", "disgusting", "awful")] fart."
 			else if(SUPER_FART in mutations)
-				message = "<b>[src]</b> unleashes a [pick("loud","deafening")] fart."
-				newtonian_move(dir)
+				message = "<b>[src]</b> unleashes a [pick("loud", "deafening")] fart."
 			else
-				message = "<b>[src]</b> [pick("passes wind","farts")]."
+				message = "<b>[src]</b> [pick("passes wind", "farts")]."
 			m_type = 2
 
 			var/turf/location = get_turf(src)
-			var/aoe_range=2 // Default
 
 			// Process toxic farts first.
 			if(TOXIC_FARTS in mutations)
-				for(var/mob/M in range(location,aoe_range))
-					if(M.internal != null && M.wear_mask && (M.wear_mask.flags & AIRTIGHT))
+				for(var/mob/living/carbon/C in range(location, 2))
+					if(C.internal != null && C.wear_mask && (C.wear_mask.flags & AIRTIGHT))
 						continue
-					// Now, we don't have this:
-					//new /obj/effects/fart_cloud(T,L)
```

Comment:
On Goon, this is just a visual yellow cloud that adds jenkem to you; we just do that, below, invisibly, without the cloud (and on Goon, the cloud has no actual icon, so it's technically invisible).

Reply:
Having a cloud would be hilarious, but I think I can see why we wouldn't have one